includes/base/element-base.php

```diff
@@ -461,6 +461,17 @@ public function add_link_attributes( $element, array $url_control, $overwrite =
 			$attributes['rel'] = 'nofollow';
 		}
 
+		if ( ! empty( $url_control['custom_attributes'] ) ) {
+			// Custom URL attributes should come as a string of comma-delimited key|value pairs
+			$custom_attributes = explode( ',', $url_control['custom_attributes'] );
```

Comment:
It should work like element custom attributes, each attribute in a separate line.

Reply:
According to the plan, it should be in a single line